packages/design-system/src/styles/components/_Button.scss

```diff
@@ -13,40 +13,46 @@
   font-weight: var(--fontWeight, $button__font-weight);
   line-height: 1.3;
   text-align: center;
-  text-decoration-color: transparent;
```

Comment:
I changed this CSS rule to target `text-decoration` instead and moved it to target `<a>` specifically.